src/base.html

```diff
@@ -42,16 +42,16 @@
       <!-- Page description -->
       {%- if page and page.meta and page.meta.description -%}
         <meta name="description"
-            content="{{ page.meta.description }}" />
+            content="{{ page.meta.description | e }}" />
       {%- elif config.site_description -%}
-        <meta name="description" content="{{ config.site_description }}" />
+        <meta name="description" content="{{ config.site_description | e }}" />
       {%- endif -%}
 
       <!-- Redirect -->
       {%- if page and page.meta and page.meta.redirect -%}
         <script>
           var anchor = window.location.hash.substr(1)
-          location.href = '{{ page.meta.redirect }}' +
+          location.href = {{ page.meta.redirect | tojson | safe }} +
```

Comment:
What exactly is happening here? I've been testing your changes but couldn't come up with a case to test drive this.

Reply:
I was going through the mkdocs website and saw the `tojson` filter: https://www.mkdocs.org/user-guide/custom-themes/#tojson and decided to use it. It probably isn't much of an issue, as you wouldn't want to use single quotes in a URL anyway, but it might fix some edge cases.